backend/worker.py

```
@@ -0,0 +1,46 @@
+    connection = pika.BlockingConnection(params)
+    channel = connection.channel()
+
+    channel.queue_declare(queue='glaucoma_queue')
+    
+    print(" [*] Connected! Listening on 'glaucoma_queue'...")
+    print("=====================================================")
+
+    channel.basic_consume(queue='glaucoma_queue', on_message_callback=callback, auto_ack=True)
+    channel.start_consuming()
+    
+except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)